cs/src/core/Index/FasterLog/FasterLog.cs

```diff
@@ -241,27 +241,28 @@ public ValueTask<long> EnqueueAsync(byte[] entry, CancellationToken token = defa
                 return new ValueTask<long>(logicalAddress);
 
             return SlowEnqueueAsync(this, entry, token);
-            static async ValueTask<long> SlowEnqueueAsync(FasterLog @this, byte[] entry, CancellationToken token)
+        }
+
+        private static async ValueTask<long> SlowEnqueueAsync(FasterLog @this, byte[] entry, CancellationToken token)
```

Comment:
Any reason for this code to change?

Reply:
I am embedding FASTER as source code into net462 project, so this is one of the places where I have to manually "fix" to be able to compile and there is no difference from performance standpoint (one can also say - personal preference).